nibabel/openers.py

```diff
@@ -14,6 +14,13 @@
 import sys
 from os.path import splitext
 
+# is indexed_gzip present?
+try:
+    from indexed_gzip import SafeIndexedGzipFile
+    HAVE_INDEXED_GZIP = True
+except:
```

Comment:
`except ImportError`

Reply:
Yikes - I got bitten by this the other week - i learnt the hard way that `except:` will also catch `SystemExit`!